static/styles/portico/portico.css

```diff
@@ -1377,7 +1376,7 @@ input.new-organization-button {
         .sidebar {
             position: fixed;
             top: 59px;
-            right: calc(100vw - 50px);
+            right: calc(100% - 50px);
```

Comment:
Explained in the commits as well as in the conversation as the browser scroll bar width should not be considered while calculating the right position.